river/bandit/BayesUCB.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+import random
+
+from river import bandit
+
+from scipy.special import btdtri
+
+
+class Beta:

```

Comment:
Thanks for reminding me! I've uploaded a new version using the _Beta_ class of the River. It can run on my computer, but when I try to pull it, there's an error in code checking :
> river/bandit/bayes_ucb.py:52: error: Cannot instantiate abstract class "Beta" with abstract attribute "cdf"  [abstract]
I'll appreciate it if you can give me some guidance :)